reVX/setbacks/structure_setbacks.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+"""
+Compute setbacks exclusions
+"""
+import os
+import re
+import logging
+
+from reVX.setbacks.base import BaseSetbacks
+
+logger = logging.getLogger(__name__)
+
+
+class StructureSetbacks(BaseSetbacks):
```

Comment:
nice, these look like really clean + simple concrete subclasses. A sign of a good refactor!